contracts/src/v0.8/dev/KeeperRegistryBase.sol

```diff
@@ -74,6 +75,8 @@ abstract contract KeeperRegistryBase is ConfirmedOwner, ExecutionPrevention, Ree
   error InvalidRecipient();
   error InvalidDataLength();
   error TargetCheckReverted(bytes reason);
+  error OnlyNonPausedUpkeep();
```

Comment:
nitpick NonPaused feels a bit like a double negative. Can we call this `OnlyActiveUpkeeps`?

Reply:
+1. I think there's already an error for active (not cancelled) upkeeps. We can try to merge some concepts of pause and cancelled probably

Reply:
Done